docker/docker-compose.dev.yml

```diff
@@ -73,7 +77,9 @@ services:
     extra_hosts:
       - "localhost:host-gateway"
     environment:
-      - DOPPLER_TOKEN=${DOPPLER_TOKEN_JOB_CREATOR}
+      - WEB3_PRIVATE_KEY=${JOB_CREATOR_PRIVATE_KEY}
+      - API_HOST=${API_HOST}
```

Comment:
Could we check if we need this? We moved `API_HOST` to the embedded configs a little while back  (https://github.com/Lilypad-Tech/lilypad/pull/253), and this may be a leftover in Doppler that we didn't get around to removing.

Reply:
Sure thing, I'll double check if we need this

Reply:
I'd also suggest *not* setting `API_HOST` unless you specifically want it (calls will just http retry to a non existant end point if you're not running the api sever ... it adds time to our integration tests amongst other things).

Reply:
Looks like this is not needed, so got rid of it